manager/probe.go

```diff
@@ -307,6 +314,16 @@ func (p *Probe) init() error {
 		p.KProbeMaxActive = p.manager.options.DefaultKProbeMaxActive
 	}
 
+	// Default retry
+	if p.ProbeRetry == 0 {
```

Comment:
It could be nice to add something to disable the retry mechanism

Reply:
Default set to 1 👍